airflow/ui/src/components/TriggerDag/TriggerDAGForm.tsx

```diff
@@ -0,0 +1,180 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Input, Button, Box, Text, Spacer, HStack } from "@chakra-ui/react";
+import { autocompletion } from "@codemirror/autocomplete";
+import { json } from "@codemirror/lang-json";
+import { githubLight, githubDark } from "@uiw/codemirror-themes-all";
+import CodeMirror, { lineNumbers } from "@uiw/react-codemirror";
+import { useEffect, useState } from "react";
+import { useForm, Controller } from "react-hook-form";
+
+import { useColorMode } from "src/context/colorMode";
+
+import { Accordion } from "../ui";
+import type { DagParams } from "./TriggerDag";
+
+type TriggerDAGFormProps = {
+  dagParams: DagParams;
+  onClose: () => void;
+  onTrigger: (updatedDagParams: DagParams) => void;
+  setDagParams: React.Dispatch<React.SetStateAction<DagParams>>;
+};
+
+const TriggerDAGForm: React.FC<TriggerDAGFormProps> = ({
+  dagParams,
+  onTrigger,
+  setDagParams,
+}) => {
+  const [jsonError, setJsonError] = useState<string | undefined>(undefined);
+  const {
+    control,
+    formState: { isDirty },
+    handleSubmit,
+    reset,
```

Comment:
I just noticed if I use the form and trigger and open it for the same DAG again, the values are kept from first run.

Can you ensure it is reset once it is re-opened (or reset post trigger on close)

Reply:
sure, let me reset the params to initial.